docs/source/_themes/sphinx_rtd_theme/static/css/theme.css

```diff
@@ -3525,6 +3525,10 @@ h6 {
   font-size: 100%;
 }
 
+.rst-content .toctree-wrapper p.caption {
```

Comment:
this didn't work and I didn't go after figuring out why, css is massive. Leaving as is.